### DISCUSSION TITLE
Tab bar on bottom

### DISCUSSION THREAD

**Discussion by FormalSnake** [ghostty-org/ghostty#3904] [Feature Requests, Ideas] [answered]
In iTerm2, the tabs are at the bottom of the screen, and the window controls (in this case traffic lights) only appeared on hover. It is not the most native but it is extremely practical that way.

**Comment by weirdan** [comment]
It's available in Linux build: https://ghostty.org/docs/config/reference#gtk-tabs-location

**Reply by FormalSnake** [reply]
yeah but i'm on mac, thanks anyway

**Reply by mitchellh** [reply, marked as answer]
There is another issue for this. We can't do this without a custom tab bar. Not planned for now, but long term yes.

**Reply by FormalSnake** [reply]
> There is another issue for this. We can't do this without a custom tab bar. Not planned for now, but long term yes.

That's perfect, thank you!
